src/components/Settings/Notifications/NotificationsDiscord.tsx

```diff
@@ -13,14 +13,16 @@ const messages = defineMessages({
   save: 'Save Changes',
   saving: 'Saving…',
   agentenabled: 'Enable Agent',
+  botUsername: 'Bot Username',
+  botAvatarUrl: 'Bot Avatar URL',
   webhookUrl: 'Webhook URL',
   webhookUrlPlaceholder: 'Server Settings → Integrations → Webhooks',
   discordsettingssaved: 'Discord notification settings saved successfully!',
   discordsettingsfailed: 'Discord notification settings failed to save.',
   testsent: 'Test notification sent!',
   test: 'Test',
   notificationtypes: 'Notification Types',
-  validationWebhookUrl: 'You must provide a valid URL',
+  validationUrl: 'You must provide a valid URL',
```

Comment:
This change seems unnecessary? Translators will have to revisit the string.

Reply:
I wanted to use the same string for both URL fields, instead of adding a new string.  I could keep it as `validationWebhookUrl`, but I was afraid that a translator might see the name and translate it as "You must provide a valid webhook URL," which wouldn't work for the avatar URL field.

Reply:
If you think it's better to add a new string `valdationBotAvatarUrl`, we can do that as well.  Just let me know what you think is the better option here!

Reply:
Oh I see we are sharing it. Carry on then.